daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala

```diff
@@ -293,9 +293,19 @@ object Ref {
     final case class Name(name: PackageName) extends PackageRef {
       override def toString: String = "#" + name
     }
+
     final case class Id(id: PackageId) extends PackageRef {
       override def toString: String = id
     }
+
+    def fromString(s: String): Either[String, PackageRef] =
```

Comment:
How about we start with another letter before `n#`. It might help debugging since the eye might naturally skip easier over the `#`. What do you think?